src/components/film-movie-form/success-modal/index.js

```diff
@@ -1,13 +1,22 @@
 import { Footer, Logo } from '../../index';
 import suc from './img/group.png';
-import './success-modal.css';
-import { useEffect } from 'react';
+import './style.css';
+import { useContext, useEffect } from 'react';
+import { AppContext } from '../../context';
+
+const SuccessModal = () => {
+  const { state, dispatch } = useContext(AppContext);
 
-const SuccessModal = ({ onCloseModal }) => {
   useEffect(() => {
     document.body.style.overflow = `hidden`;
   });
```

Comment:
useEffect should have second argument.
```suggestion
  useEffect(() => {
    document.body.style.overflow = `hidden`;
  }, []);
```

Reply:
done